.github/workflows/pr-fde.yml

```diff
@@ -0,0 +1,26 @@
+name: Validate PR for Full Disk Encryption (FDE)
+
+on:
+  pull_request:
+    paths:
+    - 'attestation/full-disk-encryption/**'
+  workflow_dispatch:
+
+jobs:
+  code_scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout PR
+        uses: actions/checkout@v3
+      - name: Set up Rust action
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Run cargo check
+        run: |
+          sudo apt-get update
+          sudo apt-get install libcryptsetup-dev
+          cd attestation/full-disk-encryption/
+          cargo check
+          cargo fmt -- --check
+          cargo clippy
```

Comment:
I think `cargo deny` can be added. And `deny` need to be installed before using.